.changeset/cyan-stingrays-own.md

```diff
@@ -0,0 +1,18 @@
+---
+'@clerk/nextjs': major
+---
+
+Drop deprecations. Migration steps:
```

Comment:
@dimkl  Should we mark these as breaking?

Reply:
I thought that the major version bump should indicate that but we can also mark them as breaking changes. All the changes with `major` version are breaking.

Reply:
Merging this to continue with the cleanup and i will update the changeset if we want to somehow mark the changes as breaking.